generate_certificate.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 # Fully qualified name of the CSR object
-csr="certificatesigningrequests.v1beta1.certificates.k8s.io"
```

Comment:
I think You need to change following line as well:
```
apiVersion: certificates.k8s.io/v1beta1
```
Line 119 in generate_certificate.sh

Reply:
Good catch!